client/sites/tools/controller.tsx

```diff
@@ -50,6 +51,11 @@ export function ToolsSidebar() {
 	);
 }
 
+export function advancedTools( context: PageJSContext, next: () => void ) {
+	context.primary = <HostingFeatures />;
```

Comment:
Can we also move/rename `client/hosting/hosting-features/components` => `client/sites/components/hosting-features`? This way, when we clean up later, we don't need to touch the old location anymore.

Reply:
fixed in 2e41f39cc15e4deaad0f6964664589e2d4ce3195

Reply:
Ah, can we drop `/tools/` from the patch, i.e. move it up one level, under `client/sites/`? The subtab upsells might need to extract some shared utils from it. Hosting features are across tabs/subtabs, not just `tools`.

Reply:
Moved here 5d023b76f9e777ef2757adb410123ae5fe66eeee